docs/edsl.html

```diff
@@ -461,7 +462,7 @@ <h3 id="convolution">Convolution</h3>
 \Large
 \textcolor{red}{O[n, x, c_o]}
 \textcolor{yellow}{=}
-\textcolor{green}{\sum<em>k \sum</em>{c_i}}
+\textcolor{green}{\sum k \sum{c_i}}
```

Comment:
You need subscripts between the `\sum` & `k` and `\sum` & `{c_i}` for this to render correctly.